src/System.Net.Sockets/src/System/Net/Sockets/NetworkStream.cs

```diff
@@ -614,6 +614,9 @@ internal bool Connected
         // Returns:
         // 
         //     An IASyncResult, representing the read.
+#if !netcore50
+        override
+#endif
```

Comment:
Yes most of the .NET Standard 2.0 work is targeting "uap10.1", however that ties to a new tool chain that is also under development in TFS.

For some things we could add them to the netcore50 version of the library assuming it isn't part of the SharedLibrary and doesn't require any new dependent APIs that are in contracts that are part of the SharedLibrary. My expectation is that most of the .NET Standard 2.0 will require newer versions of things like System.Runtime which means we won't really be able to support them in netcore50 and we will need to use uap10.1 instead.